src/main/java/frc/robot/Orchestrator.java

```diff
@@ -191,6 +204,10 @@ public Command indexBasic() {
         .withTimeout(1);
   }
 
+  public Command stopFlywheel() {
+    return Commands.run(() -> shooter.manualShoot(0));
```

Comment:
This does nothing - it runs a method that returns the Command, and doesn't run the command.

It should just be: `return shooter.manualShoot(0);`

Reply:
Oops

Reply:
true. I was thinking manualShoot was a runnable, didnt realize it was a command